.zuul.yaml

```diff
@@ -19,75 +19,20 @@
       # Donot use openstack services containers from meta content provider master
       # job.
       cifmw_update_containers_openstack: false
-      # controlplane customization to deploy telemetry service
-      cifmw_edpm_prepare_timeout: 60
-      cifmw_edpm_prepare_kustomizations:
-        - apiVersion: kustomize.config.k8s.io/v1beta1
-          kind: Kustomization
-          namespace: openstack
-          patches:
-          - patch: |-
-              apiVersion: core.openstack.org/v1beta1
-              kind: OpenStackControlPlane
-              metadata:
-                name: unused
-              spec:
-                telemetry:
-                  enabled: true
-                  template:
-                    metricStorage:
-                      enabled: true
-                      monitoringStack:
-                        alertingEnabled: false
-            target:
-              kind: OpenStackControlPlane
-      cifmw_install_yamls_whitelisted_vars: &install_yamls_whitelist
-        - 'WATCHER_REPO'
-        - 'WATCHER_BRANCH'
-        - 'OUTPUT_DIR'
-      pre_deploy_create_coo_subscription:
-        - name: Deploy cluster-observability-operator
-          type: playbook
-          source: "{{ watcher_coo_hook }}"
-      post_deploy:
-        - name: Deploy watcher service
-          type: playbook
-          source: "{{ watcher_hook }}"
-          extra_vars:
-            watcher_catalog_image: "{{ content_provider_registry_ip }}:5001/openstack-k8s-operators/watcher-operator-index:{{ zuul.patchset }}"
-      cifmw_tempest_tempestconf_config:
-        overrides: |
-          compute.min_microversion 2.56
-          compute.min_compute_nodes 2
-          placement.min_microversion 1.29
-          compute-feature-enabled.live_migration true
-          compute-feature-enabled.block_migration_for_live_migration true
-          service_available.sg_core true
-          telemetry_services.metric_backends prometheus
-          telemetry.disable_ssl_certificate_validation true
-          telemetry.ceilometer_polling_interval 15
-          optimize.datasource ''
-      cifmw_test_operator_tempest_external_plugin:
-        - repository: "https://opendev.org/openstack/watcher-tempest-plugin.git"
-          changeRepository: "https://review.opendev.org/openstack/watcher-tempest-plugin"
-          changeRefspec: "refs/heads/master"
+      deploy_watcher_service_extra_vars:
+        watcher_catalog_image: "{{ content_provider_registry_ip }}:5001/openstack-k8s-operators/watcher-operator-index:{{ zuul.patchset }}"
+      cifmw_extras:
+        - "@{{ ansible_user_dir }}/{{ zuul.projects['github.com/openstack-k8s-operators/ci-framework'].
+           src_dir }}/scenarios/centos-9/multinode-ci.yml"
+        - "@{{ ansible_user_dir }}/{{ zuul.projects['github.com/openstack-k8s-operators/ci-framework'].
+           src_dir }}/scenarios/centos-9/horizon.yml"
```

Comment:
Horizon was enabled there already https://github.com/openstack-k8s-operators/ci-framework/blob/8eab954e13cde6c3225e9606d9c9df5b273e6a10/zuul.d/edpm_multinode.yaml#L319 that's why it is added here again.